FRC1710-2019/src/main/java/frc/robot/Robot.java

```diff
@@ -40,8 +42,11 @@ public void robotInit() {
   public void teleopPeriodic() {
     //This makes the robot drive | Turn power is multiplied by .3 to make it slower and drive is by .5 to make is slower as well
    Drive.arcadeDrive(-Drive.getTurnPower() * .2, Drive.getForwardPower() * .35);
+   Ballmech.runBallMech();
```

Comment:
Looking at the following code you have here, it looks like you are doing all 3 things (if I did not know/understand the insides of how Ballmech.java worked). 

1. The ball mech will run.
2. The ball mech will undo.
3. The ball mech will stop if there is a problem??? (That last one could perhaps use a better name also).

This could get confusing for someone who does not understand or has not read how the Ballmech.java file is written.
Do you think it might be better to consolidate this logic into the class itself and only call one method like `Ballmech.teleop()` or `Ballmech.run()`?

For example
```
public void teleopPeriodic() {
 ...
 Ballmech.teleop();
...
}
```

Then inside of Ballmech.java have conditional logic/joystick checking to determine what to do?